microprofile.jdt/com.redhat.microprofile.jdt.quarkus/src/main/java/com/redhat/microprofile/jdt/internal/quarkus/providers/QuarkusConfigRootProvider.java

```diff
@@ -318,10 +318,22 @@ private void processConfigGroup(String extensionName, IJavaElement javaElement,
 					} else {
 						subKey = baseKey + "." + name;
 					}
-					final String defaultValue = configItemAnnotation == null ? ConfigItem.NO_DEFAULT
-							: getAnnotationMemberValue(configItemAnnotation, "defaultValue");
-
+					 
 					String fieldTypeName = getResolvedTypeName(field);
+					
+					String defaultValue;
+					if (configItemAnnotation != null && QuarkusConstants.CONFIG_ITEM_ANNOTATION.indexOf(configItemAnnotation.getElementName()) != -1) {
+						defaultValue = getAnnotationMemberValue(configItemAnnotation, "defaultValue");

```

Comment:
Define in QuarkusConstants 

`public static final String CONFIG_ITEM_ANNOTATION_DEFAULT_VALUE = "defaultValue";` and use it. You can do that for other annotation member value (like "name"), etc